runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpStateSingle.java

```diff
@@ -248,6 +252,8 @@ XpSnapshotSingle snapshot()
 			.actionsRemainingToGoal(getActionsRemaining())
 			.actionsPerHour(getActionsHr())
 			.timeTillGoal(getTimeTillLevel())
+			.startLevelLabel("Lvl. " + startLevel)
```

Comment:
I would rather propagate startLevelXp and endLevelXp and create labels from UI, I dont think creating of labels should be located here